packages/react/src/Bento/Bento.module.css

```
@@ -0,0 +1,285 @@
+.Bento {
+  display: grid;
+  width: 100%;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  grid-template-rows: minmax(40px, auto);
```

Based on Enterprise designs, where some items are ~560px or ~640px tall I'd go with a default value of 5rem. As it's flexible enough to get those values by just spanning the item rows.

- 5rem x 7 rows = 560px
- 5rem x 8 rows = 640px

+ I think we could just use `grid-auto-rows`

grid
```suggestion
  grid-auto-rows: var(--base-size-80);
```

---

I tried `--base-size-80` which looked a bit to large but `--base-size-40` was just right. Is this update alright to make or should I leave 80. I think creating a demo with content will help in the docs greatly.

---

I got this working as the design intended with the --base-size-80! I just had to work with it a bit!